File: Droid/07_Classes/src/Pessoa.kt
class Pessoa(private var _nome: String, var anoNascimento: Int) {

    var idade: Int
        get() = (2026 - this.anoNascimento)

        set(value) {
            this.anoNascimento = 2026 - value
        }

    var nome: String
        get() = this._nome

        private set(value) {

            this._nome = value
        }



    /*
    fun calcIdade(): Int{
        return 2026 - this.anoNascimento

    }
     */
}



// bla bla bla :Lista

File: iOS/05_classes/05_classes/Pessoa.swift
//
//  Pessoa.swift
//  05_classes
//
//  Created by Gonçalo Feliciano on 24/02/2026.
//



class Pessoa {
    
    var nome:String
    var idade:Int
    
    init(nome: String, idade: Int = 0) {
        self.nome = nome // self -> this
        self.idade = idade
    }
    
    
    func getNome() {
        print(self.nome)
    }
    
    func setNome(nome:String) {
        self.nome = nome
    }
    
}
